debian/control

```diff
@@ -44,7 +44,9 @@ Description: Intel(R) XPU System Management Interface(xpu-smi) is a free and
 Package: xpu-smi
 Architecture: amd64
 Depends:
- libxpum1,
+ libze1 (>= 1.17.6),
```

Comment:
@mckees why do we want these dependencies here now ? are they direct dependencies of xpu-smi ?

Reply:
without those pkgs installed on an intel system ..
xpu-smi is useless

Reply:
got it but are they direct dependencies or not ? if they are direct dependencies of libxpum, no need to specify them here

Reply:
they are not necessarily build-time dependencies, but they are runtime dependencies, there's a difference there

Reply:
@androniychuk :) sorry but that does not answer my question : direct dependency (runtime or not, whatever) ?

Reply:
@androniychuk :) sorry but that does not answer my question : direct dependency (runtime or not, whatever) ?

Reply:
That is what i have in mind :
```
A() {
  B()
}
B() {
  C()
}
B is direct dependency of A
```

```
A() {
  B()
  C()
}
B & C are direct dependencies of A
```

Does that make sense ?

Reply:
> without those pkgs installed on an intel system .. xpu-smi is useless runtime dependencies...

here is my previous response, this is a more detailed description of the dependencies
runtime dependencies, they need to be there in order for xpu-smi to work properly.
They are not dependencies of another pkg that is a dependency of xpu-smi.

Reply:
Seems that i cannot be clear enough to make you understand my point, i will not block the PR for that if @mckees is fine with that, i will approve